CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # CHANGELOG
 
+## 2.0.0 (2022-11-14)
+
+MLflow 2.0.0 includes several breaking changes from the 1.x API, additional major features and improvements
```

Comment:
Can we add some excitement here, e.g. one more sentence indicating that this release is a major milestone for MLflow that simplifies end-to-end MLOps workflows?